workflows/diagnostics/tests/offline/test_compute_diagnostics.py

```diff
@@ -1,143 +1,6 @@
-import numpy as np
-import pytest
 import xarray as xr
 
-from fv3net.diagnostics.offline.compute_diagnostics import (
-    weighted_average,
-    _snap_mask_to_type,
-    _snap_net_precipitation_to_type,
-    _conditional_average,
-    merge_diagnostics,
-)
-
-
-da = xr.DataArray(np.arange(1.0, 5.0), dims=["z"])
```

Comment:
I can't remember if there was any good reason why the `vcm.weighted_average` tests were here and not in vcm, but I moved them.